src/custodian/src/custodian_util.erl

```diff
@@ -43,7 +43,7 @@ report() ->
     fold_dbs([], Fun).
 
 ensure_dbs_exists() ->
-    DbName = config:get("mem3", "shards_db", "dbs"),
+    DbName = config:get("mem3", "shards_db", "_dbs"),
```

Comment:
Any reason to not use: https://github.com/apache/couchdb/blob/3.x/src/mem3/src/mem3_sync.erl#L315-L316

Reply:
Other than laziness, the only reason is that I was recently chastised by @rnewson for using a function in a different module that was not in an `_util` module. However, I do prefer keeping things DRY, so I've changed it!